db/repos/accounts.js

```diff
@@ -359,7 +351,7 @@ class AccountsRepo {
 				options.sortField.map((field, index) => {
 					sortSQL.push(this.pgp.as.format('$1:name $2:raw', [field, options.sortMethod[index]]));
 				});
-				sql = sql + `ORDER BY ${sortSQL.join(', ')}`;
```

Comment:
@vitaly-t The case handles the situation where you sort on multiple fields. 
```
Select * from mem_accounts order by address ASC, isDelegate DESC;
```
Removing that `, ` will break the query.

Reply:
I only removed `, `, because that's the default `.join`, and you do not need an extra space there ;)

Reply:
If its default for `join` then its ok. I was not aware that `join` have any default.